go/cmd/topo2topo/topo2topo.go

```diff
@@ -64,6 +67,14 @@ func main() {
 
 	ctx := context.Background()
 
+	if *compare {
+		compareTopos(ctx, fromTS, toTS)
+		return
+	}
+	copy(ctx, fromTS, toTS)
+}
+
+func copy(ctx context.Context, fromTS, toTS *topo.Server) {
```

Comment:
Minor naming nit -- if you're calling the other function `compareTopos` this should be `copyTopos` for symmetry.